client/my-sites/plans/jetpack-plans/slug-to-selector-product.ts

```diff
@@ -141,8 +142,8 @@ function itemToSelectorProduct(
 			yearlyProductSlug = PRODUCTS_LIST[ item.product_slug as JetpackProductSlug ].type;
 		}
 
-		// We do not support TERM_BIENNIALLY for Jetpack plans
-		if ( item.term === TERM_BIENNIALLY ) {
+		// We do not support TERM_BIENNIALLY or TERM_TRIENIALLY for Jetpack plans
+		if ( item.term === ( TERM_BIENNIALLY || TERM_TRIENNIALLY ) ) {
```

Comment:
I don't think this statement will do what you want it to; since `TERM_BIENNIALLY` is truthy, it will evaluate to `item.term === TERM_BIENNIALLY`.

Maybe:
```suggestion
		if ( [ TERM_BIENNIALLY, TERM_TRIENNIALLY ].includes( item.term ) ) {
```

Reply:
Thanks! 👍